lib/utils/interactiveProgressHanlder.js

```diff
@@ -34,7 +34,7 @@ function getProgressExplanation(status, code = '') {
 function handleProgress(status, code) {
 	const text = getProgressExplanation(status, code);
 
-	text && console.log(stColors.suit(`  - ${text}`));
+	text && logger.info(text);
 }
```

Comment:
Forgot dashes and spaces here ?
`  - ${text}`

Reply:
I removed this intentionally, because it's now displayed with "Info: " prefix.

Reply:
should be mentioned in ticket I think, because that was request form qa team

Reply:
ok. Will add a comment there